src/features/profile/components/Header/NonBeneficiaryHeader/NonBeneficiaryHeader.web.test.tsx

```diff
@@ -181,21 +183,28 @@ describe('<NonBeneficiaryHeader/>', () => {
     it('should render the younger badge for user under 18 years old', () => {
       const today = '2021-01-30T00:00:00Z'
       mockdate.set(new Date(today))
+
       const { getByTestId } = render(
         <NonBeneficiaryHeader
           eligibilityStartDatetime="2021-01-31T00:00Z"
           eligibilityEndDatetime="2022-01-31T00:00Z"
           isEligibleForBeneficiaryUpgrade={false}
         />
       )
+
       getByTestId('younger-badge')
     })
   })
 
   describe('<React.Fragment />', () => {
-    it('should not display banner or badge if isEligibleForBeneficiaryUpgrade is false and user not under 18 years old', () => {
+    it('should not display banner or badge if the user is over 15 years old but whithout nextIdentityStep or pendingIndentityCheck (non eligible to credit)', () => {
```

Comment:
l'ajout de `(non eligible to credit)` me confond au lieu de m'éclairer, ce qui n'est certainement pas ton intention.

Est-ce que tu peux retourner la phrase pour expliciter d'abord le métier ?

`it('should not display banner or badge if the user is over 15 years old and not eligible to credit (no nextSubscriptionStep and  no identityCheckPending))`

Qu'est-ce que tu en dis ?

Reply:
Autre remarque : 

On a décrété assez récemment que la marche à suivre pour la duplication des tests en .web n'était nécessaire que quand elle testait un comportement réellement différent sur le web.

Exemple : on veut un affichage spécifique de composant sur le web, ou bien une implémentation diffère réellement selon la plateforme (tu peux trouver des exemples dans le code avec l'utilisation de `Platform.OS`).

Ici, le comportement n'a pas de raison d'être différent, donc le test semble être dupliqué sans raison. Même si ça n'est pas ton code originellement, tu peux le supprimer et t'en tenir au test natif.